### ISSUE TITLE
Loading a construction after loading construction

### ISSUE THREAD

**Issue by dickinson0718** [dulimarta/spherical-easel#84]
1) Load a public construction (like "Venn Diagram") it appears and displays correctly (labels and all!).
2) Load another public construction (like "Construction demo") It also loads and displays correctly.  
3) Load a private construction (like "Test Polygon 5") and the construction doesn't load and the display is broken
![Screenshot 2024-07-25 at 2 13 18 PM](https://github.com/user-attachments/assets/a4477797-93d0-40e9-b4c6-5c2a9a766ec3)
Moving or rotating doesn't fix the issue
![Screenshot 2024-07-25 at 2 14 56 PM](https://github.com/user-attachments/assets/455b8427-7bc9-4051-b946-5aacf8562c3e)

However, starting from a fresh reload "Test Polygon 5" loads ok (subject of issue #83 ) but then after a move displayed correctly.

**Comment by dulimarta** [comment]
Apparently, the `parse()` function of every `xxxxCommand.ts` incorrectly uses Vector3 `copy()` to initialize `SELabel` position, when `positionVector` is a property with custom `set` and `get` functions.

```
myLabel.positionVector.copy(newLoc)   // incorrect
myLabel.positionVector = newLoc       // corrrect
```

**Comment by dulimarta** [comment]
The issue originates from the `removeAllFromLayers` function of `stores/se.ts`, specifically with cleaning up `seSegments`.

Apparently, there is a misspelling of variable name in `init()` function of `stores/se.ts`.

```
seSegmentMap.clear()
seSegments.value.splice(0) // <== this is the offending line
```

The correct code should be:
```
seSegmentMap.clear()
seSegmentIds.value.splice(0)
```

This issue is fixed in commit `95db600b0`

**Comment by dickinson0718** [comment]
While investigating an issue with not being able to select segments I noticed the following in `Highlighter.ts`

```
    this.hitSEPoints.clear();
    this.hitSELines.clear();
    this.hitSESegments.clear();
    this.hitSECircles.clear();
    this.hitSEEllipses.clear();
    this.hitSELabels.clear();
    this.hitSEAngleMarkers.clear();
    this.hitSEParametrics.clear();
    this.hitSEPolygons.clear();
    this.infoText.hide();
```

Should these clear statements be removed?
